townhall/myapi/dao.py

```diff
@@ -85,4 +85,12 @@ def delete_opportunity(opportunity_id: int):
         try:
             Opportunity.objects.get(id=opportunity_id).delete()
         except Opportunity.DoesNotExist:
-            pass
\ No newline at end of file
+            pass
+
+class OrganizationDao:
+    def get_organization(id: int) -> Organization:
```

Comment:
Type hinting here is a bit non-precise since this method can return None, to rectify this:
Put this at the very top of the file `import typing`
This method type hint should be `typing.Optional[Organization]` which signifies that this method is not guaranteed to return an Organization

I think theres other places in the file where this needs to be fixed in get methods, if you can adjust them there too would be great thanks